src/main/java/seedu/address/logic/parser/schedule/AddEventCommandParser.java

```diff
@@ -8,6 +8,7 @@
 import static seedu.address.logic.parser.CliSyntax.PREFIX_EVENT_DESCRIPTION;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_RECUR_FREQUENCY;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_TIME;
+import static seedu.address.model.recurfrequency.RecurFrequency.DEFAULT_RECURRENCE;
```

Comment:
I just realised RecurFrequency is in other package. Since there is only one class and it is part of an Event, I think it should be part of the Schedule package.

Reply:
Sure. Should I move the Tag out of the package too?

Reply:
And put it into the person package? Sure.

Reply:
Will we remove the tag package? Or move the entire tag package inside person package instead?

Reply:
Done!